app/jobs/kit_import_job.rb

```diff
@@ -17,6 +18,12 @@ def perform(file_or_folder, logger:, user_id: nil)
     @logger = logger
     @project = nil
     @report_templates_dir = Configuration.paths_templates_reports
+    @templates_dirs = TEMPLATE_TYPES.map do |template_type|
+      [
+        template_type,
+        Pathname.new(Configuration.send("paths_templates_#{template_type}"))
+      ]
```

Comment:
If you want to 🤓 out, you can do this in a single SQL query, by prebuilding the names from the TEMPLATE_TYPEs.